Cargo.toml

```diff
@@ -1603,6 +1603,15 @@ description = "Demonstrates how the AlignItems and JustifyContent properties can
 category = "UI (User Interface)"
 wasm = false
 
+[[example]]
+name = "text_wrap"
+path = "examples/ui/text_wrap.rs"
+
+[package.metadata.example.text_wraap]
```

Comment:
Little typo here throwing the CI, should be `[package.metadata.example.textwrap]`